CocoaLumberjack.podspec

```diff
@@ -32,6 +32,12 @@ Pod::Spec.new do |s|
 
   s.subspec 'Swift' do |ss|
     ss.dependency 'CocoaLumberjack/Core'
-    ss.source_files = 'Sources/CocoaLumberjackSwift/**/*.swift', 'Sources/CocoaLumberjackSwiftSupport/include/**/*.{h}'
+    ss.source_files = 'Sources/CocoaLumberjackSwift/**/*.swift',  'Sources/CocoaLumberjackSwiftLogBackend/**/*.swift', 'Sources/CocoaLumberjackSwiftSupport/include/**/*.{h}'
+  end
+
+  s.subspec 'SwiftLog' do |ss|
+    ss.dependency 'SwiftLogAPI'
+    ss.dependency 'CocoaLumberjack/Swift'
```

Comment:
That is incorrect. The SwiftLog backend only depends on `Core`. The `Swift` subspec only provides a Swift-Interface over `Core` (as in Swift functions for the ObjC-Macros that aren't imported into Swift).